pymess/models/push.py

```diff
@@ -63,15 +63,22 @@ class Meta(BaseRelatedObject.Meta):
 class AbstractPushNotificationTemplate(BaseAbstractTemplate):
 
     heading = models.TextField(verbose_name=_('heading'))
+    trigger_action_alternative = models.TextField(verbose_name=_('trigger action alternative'), null=True, blank=True)
 
     def get_controller(self):
         from pymess.backend.push import PushNotificationController
         return PushNotificationController()
 
     def send(self, recipient, context_data, related_objects=None, tag=None, **kwargs):
+        extra_data = kwargs.pop('extra_data', {})
+        if self.trigger_action_alternative:
+            extra_data.setdefault(
+                'redirectUrl', self.render_text_template(self.trigger_action_alternative, context_data, recipient)
```

Comment:
This doesn't make sense when redirectUrl is attribute for one push notification backend